source/_integrations/honeywell.markdown

```diff
@@ -48,7 +48,40 @@ The climate platform integrates Honeywell US-based thermostats into Home Assista
 
 All [climate services](/integrations/climate) are supported except set_swing_mode.
 
-There is a "known" issue related to setting the temperature from Home Assistant with some thermostats. If your instance receives errors when setting the temperature above or below some value, go to the Honeywell web page and set the temperature to the max or min for the mode you are using, then adjust from Home Assistant.
+Due to the instability of the Honeywell total connect system, service calls within automations should repeat until success similar to the following example:
+
+```yaml
+alias: 15 No one home heat
+description: Everyone has left home
+trigger:
+  - platform: numeric_state
+    entity_id: zone.home
+    for:
+      hours: 0
+      minutes: 10
+      seconds: 0
+    below: 1
+action:
+  - repeat:
+      sequence:
+        - service: climate.set_temperature
+          continue_on_error: true
```

Comment:
That sounds like bad practice?

Reply:
Do we have a better/correct way to perform retries when the server is "busy" and rejects the request?

Reply:
NBope, we do not have that, however, this is not something we should recommend on using.

Reply:
```suggestion
```